go.mod

```diff
@@ -103,7 +102,7 @@ require (
 	github.com/go-openapi/jsonreference v0.19.5 // indirect
 	github.com/go-openapi/swag v0.19.14 // indirect
 	github.com/go-stack/stack v1.8.0 // indirect
-	github.com/gofrs/uuid v4.0.0+incompatible // indirect
+	github.com/gofrs/uuid v4.2.0+incompatible
```

Comment:
This should go in the direct dependencies.

Has `go mod tidy -compat=1.17` been run? Does it change the file?

Reply:
Let me check.